.pre-commit-config.yaml

```diff
@@ -1,12 +1,8 @@
 repos:
--   repo: https://github.com/ambv/black
-    rev: stable
-    hooks:
-    - id: black
-      language_version: python3.7
-      args: [--check]
 -   repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v1.2.3
     hooks:
     - id: flake8
+      additional_dependencies: ['flake8-black', 'flake8-bugbear', 'flake8-docstrings', 'flake8-quotes']
       args: [--config=.flake8]
+      exclude: (__pycache__)|(build)|(^scripts)
```

Comment:
In order to *exactly* match what's in the .flake8 file, I think each term should be surrounded by `^` and `$`.

Reply:
Can you also add a comment line right above this to explain why we're having to do this both here and in the config file?

Reply:
Discussed IRL: The regex in the `exclude` field behaves weirdly, so adding the anchor doesn't match the desired files correctly.